CONTRIBUTING.rst

```diff
@@ -28,11 +28,18 @@ Code Style
   rule is designed to keep code readable without forcing excessive wrapping for
   more deeply nested control flow constructs.
 
-* We prefer single quoted strings except in JSON literals.
+* We prefer single quoted string literals. We used to use double quotes in JSON
+  literals but that convention is now deprecated and all new string literals are
+  single quoted except as noted below.
```

Comment:
When writing new code around old JSON literals do we match style, or covert all to single quotes?